.github/workflows/publish.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
+      - run: npm install
+      - run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
```

Comment:
@wi3land I added this so GitHub Actions will auto-publish to npm when you create a new release on GitHub. Can you please do one of the following:

1. Add an NPM_TOKEN environment variable in the project's settings with a token from npmjs.org? Then, create a v0.9.0 release on GitHub.
2. Provide me with project admin access and publishing access on npmjs.org so I can do it myself.

Reply:
It'd be awesome if we could rename the `master` branch to `main` before this release too.